Wdvh/absolutecard/absolutecard.py

```diff
@@ -0,0 +1,456 @@
+from pathlib import Path
+from re import findall
+
+from modules.CardType import CardType
+from modules.Debug import log
+
+class StandardTitleCard(CardType):
+    """
+    This class describes a type of CardType that produces the 'generic' title
+    cards based on Reddit user /u/UniversalPolymath. This card supports 
+    customization of every aspect of the card, but does not use any arbitrary
+    data.
+    """
+
+    """Directory where all reference files used by this card are stored"""
+    REF_DIRECTORY = Path(__file__).parent / 'ref'
+
+    """Characteristics for title splitting by this class"""
+    TITLE_CHARACTERISTICS = {
+        'max_line_width': 32,   # Character count to begin splitting titles
+        'max_line_count': 3,    # Maximum number of lines a title can take up
+        'top_heavy': False,     # This class uses bottom heavy titling
+    }
+
+    """Default font and text color for episode title text"""
+    TITLE_FONT = str((REF_DIRECTORY / 'TerminalDosis-Bold.ttf').resolve())
+    TITLE_COLOR = '#FFFFFF'
+
+    """Default characters to replace in the generic font"""
+    FONT_REPLACEMENTS = {}
+
+    """Whether this CardType uses season titles for archival purposes"""
+    USES_SEASON_TITLE = True
+
+    """Standard class has standard archive name"""
+    ARCHIVE_NAME = 'standard'
```

Comment:
```suggestion
    ARCHIVE_NAME = 'Absolute Ordering Style`
```
Feel free to suggest a better name for this